app/models/spree/order.rb

```diff
@@ -605,7 +605,9 @@ def create_proposed_shipments
       adjustments.shipping.delete_all
       shipments.destroy_all
 
-      packages = OrderManagement::Stock::Coordinator.new(self).packages
+      packages = OrderManagement::Stock::Coordinator.new(self).packages(
+        checkout: true, apply_tags: true
+      )
```

Comment:
I dont think this is correct, this step is also executed when you create an order in the backoffice and so the checkout/tags variables would be wrong in that case...

Reply:
Ah okay that makes sense - I think I was focused on the case where we wanted to be able to change the shipping method on an existing order in the backend and have all methods available... forgot about creating one. I think we can still make it work, but definitely trickier.

Reply:
I think this commit a36c2bfb50962d4d6499e148db4e7950db6ab4e9 should address this.